testing/brian_testing/brian_test.py

```diff
@@ -471,6 +501,10 @@ def start_new():
             if message == "good_to_start":
                 good_to_start = True
 
+            if message == "extra_good_to_start":
+                dsp_finish_counter = 1
```

Comment:
No. I use a counter here because when you first start up you need to look ahead by 2 and once you propagate through the full system once, you only need to look ahead by one.

Reply:
OK I think this method needs  a bit more explanation, like why do you need three different flags to know that you're good to start another sequence?